task.sql

```diff
@@ -4,30 +4,56 @@ CREATE DATABASE ShopDB;
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 );
 
+CREATE TABLE Products (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Warehouses (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Address VARCHAR(50),
+    CountryID INT,
+    PRIMARY KEY (ID),
+    FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION
+);
+
 CREATE TABLE ProductInventory (
-    ID INT,
-    ProductName VARCHAR(50),
+    ID INT AUTO_INCREMENT,
+    ProductID INT,
+    WarehouseID INT,
     WarehouseAmount INT,
-    WarehouseName VARCHAR(50),
-    WarehouseAddress VARCHAR(50), 
-    CountryID INT,
-	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
-    PRIMARY KEY (ID)
+    PRIMARY KEY (ID),
+    FOREIGN KEY (ProductID) REFERENCES Products (ID) ON DELETE NO ACTION,
+    FOREIGN KEY (WarehouseID) REFERENCES Warehouses (ID) ON DELETE NO ACTION 
 );
 
+
 -- Populate test data
+-- Countries
+INSERT INTO Countries (Name)
+	VALUES ('Country1');
+INSERT INTO Countries (Name)
+	VALUES ('Country2');
+
+-- Products
+INSERT INTO Products (Name)
+    VALUES ('AwersomeProduct');
```

Comment:
There's a typo in the product name 'AwersomeProduct'. It should likely be 'AwesomeProduct'.